tests/test_main.py

```diff
@@ -23,6 +25,34 @@ def test_module_invocation() -> None:
     assert match is not None
 
 
+def test_module_invocation_version() -> None:
+    proc = subprocess.run(
+        [sys.executable, '-m', 'pyright', '--version'],
+        check=True,
+        stdout=subprocess.PIPE,
+        env=dict(os.environ, PYRIGHT_PYTHON_FORCE_VERSION='1.1.223'),
+    )
+    assert proc.returncode == 0
+    output = proc.stdout.decode('utf-8')
+    match = VERSION_REGEX.match(output)
+    assert match is not None
+    assert match.group(1) == '1.1.223'
+
+
+def test_module_invocation_latest_version() -> None:
+    proc = subprocess.run(
+        [sys.executable, '-m', 'pyright', '--version'],
+        check=True,
+        stdout=subprocess.PIPE,
+        env=dict(os.environ, PYRIGHT_PYTHON_FORCE_VERSION='latest'),
+    )
+    assert proc.returncode == 0
+    output = proc.stdout.decode('utf-8')
+    match = VERSION_REGEX.match(output)
+    assert match is not None
+    assert version.parse(match.group(1)) > version.parse('1.1.223')
```

Comment:
We should probably check that the version is greater than or equal to the auto-updated pyright version
```suggestion
    assert version.parse(match.group(1)) > version.parse(__pyright_version__)
```